Wishlist dropin (#347)
commerce-docs/microsite-commerce-storefront: src/content/docs/dropins/wishlist/containers/wishlist-toggle.mdx

---
title: WishlistToggle container
description: Learn about the WishlistToggle container.
---

import Diagram from '@components/Diagram.astro';
import OptionsTable from '@components/OptionsTable.astro';

> The WishlistToggle container provides a button which allows the user to add and remove a specific product to / from a wishlist.

Reviewer: ```suggestion
The <code>WishlistToggle</code> container provides a button for adding and removing specific products.
```

> Its behavior and look and feel is driven by configuration options such as size and variant.

Reviewer: ```suggestion
Configuration options drive its behavior and appearance, such as size and variant.
```
